src/beacon/app/dashboard/devices/page.tsx

```
@@ -0,0 +1,657 @@
+  AlertTriangle,
+  BarChart3,
+  Plus,
+  ArrowRight,
+  RefreshCw,
+  AlertCircle,
+  Clock,
+  Calendar,
+  ChevronLeft,
+  ChevronRight,
+  Layers,
+  Map
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)